### PR TITLE
repokitteh documentation

### PR DESCRIPTION
**Pull request by itayd**
*Description*:
Add documentation for RepoKitteh functionality.

*Risk Level*:
None.

*Testing*:
None.

*Docs Changes*:
This file.

*Release Notes*:
None.

### CODE REVIEW

File: source/docs/repokitteh.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# RepoKitteh
+
+## What is RepoKitteh?
+
+<img src="https://repokitteh.io/logo.svg" height="100" align="right">
+
+[RepoKitteh](https://repokitteh.io) is a [GitHub application](https://developer.github.com/apps/) that provides and easy way to create, integrate and maintain GitHub bots. It is deployed in GCP and supplied to Envoy under a contract with the CNCF.
```
Comment (htuch) [inline source/docs/repokitteh.md]: s/and/an/

File: source/docs/repokitteh.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+The application is installed on specific GitHub repositories and interacts with these by receiving webhooks and making GitHub API calls. A root `repokitteh.sky` script tells the application what to do based on the webhook received.
+
+## Integration with Envoy
+The file [repokitteh.sky](https://github.com/envoyproxy/envoy/blob/master/repokitteh.sky), which resides in the root of the Envoy repository tells RepoKitteh what functionality to use. The file is written in the [~~Skylark~~ Starlark language](https://github.com/bazelbuild/starlark/), which is a non-turing complete Python dialect.
```
Comment (htuch) [inline source/docs/repokitteh.md]: s/non-turing Python dialect/Python dialect with well defined threading and hermeticity guarantees/

File: source/docs/repokitteh.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Current Functionality
+### [Assign](https://github.com/softkitteh/repokitteh-modules/blob/master/assign.sky)
+Set assignees to issues or pull requests.
+
+Examples:
+```
+/assign @htuch
+```
+Adds @htuch as an assignee to the issue or pull request that this comment is made on.
```
Comment (htuch) [inline source/docs/repokitteh.md]: You can replace me with `@foobar` :P

File: source/docs/repokitteh.md (hunk 1 above)
Comment (itayd) [reply source/docs/repokitteh.md]: 👍

File: source/docs/repokitteh.md (hunk 2 above)
Comment (itayd) [reply source/docs/repokitteh.md]: 👍

File: source/docs/repokitteh.md (hunk 3 above)
Comment (itayd) [reply source/docs/repokitteh.md]: 👍 :(

**Pull request merged**